File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Ext/Db4oUnexpectedException.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System;

namespace Db4objects.Db4o.Ext
{
	/// <summary>Unexpected fatal error is encountered.</summary>
	/// <remarks>Unexpected fatal error is encountered.</remarks>
	[System.Serializable]
	public class Db4oUnexpectedException : Exception
	{
		public Db4oUnexpectedException(Exception cause) : base(cause.Message, cause)
		{
		}
	}
}

File: db4oj/core/src/com/db4o/ext/Db4oUnexpectedException.java
/* Copyright (C) 2007 Versant Inc. http://www.db4o.com */
package com.db4o.ext;

import com.db4o.foundation.*;

/**
 * Unexpected fatal error is encountered.
 */
public class Db4oUnexpectedException extends ChainedRuntimeException {

	public Db4oUnexpectedException(Throwable cause) {
		super(cause.getMessage(), cause);
	}

}
